image/oci/layout/oci_dest.go

```diff
@@ -312,6 +351,26 @@ func (d *ociImageDestination) CommitWithOptions(ctx context.Context, options pri
 	if err != nil {
 		return err
 	}
+	// Delete unreferenced blobs (e.g. old signature manifest and its config)
+	if !d.blobsToDelete.Empty() {
+		count := make(map[digest.Digest]int)
```

Comment:
```suggestion
		blobsUsedInRootIndex := make(map[digest.Digest]int)
```
to be explicit about intent